tests/multiprocess_gpu_test.py

```diff
@@ -38,8 +46,72 @@
 except ImportError:
   pytest = None
 
+config.update("experimental_xmap_spmd_lowering", True)
 config.parse_flags_with_absl()
 
+X_SIZE = 4
+Y_SIZE = 2
+Z_SIZE = 2
+ALL_AXES = ("x", "y", "z")
+
+def jax_multiprocess_init():
```

Comment:
Please don't name these functions as the generic thing. This is for slurm so name it `slurm_multiprocess_init`.

Reply:
that's reasonable, will rename